datasets/covid19_nyt/pipelines/load_from_github/pipeline.yaml

```diff
@@ -0,0 +1,224 @@
+
+      Specifically, each participant was asked: How often do you wear a mask in public when you expect to be within six feet of another person?
+
+      This survey was conducted a single time, and at this point we have no plans to update the data or conduct the survey again.
+
+dag:
+  airflow_version: 2
+  initialize:
+    dag_id: load_from_github
+    default_args:
+      owner: Google
+      depends_on_past: False
+      start_date: "2022-06-08"
+    max_active_runs: 1
+    schedule_interval: "@daily"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: BashOperator
+      description: Task to download and concat `us-counties-YYYY.csv` files
+      args:
+        task_id: download_csv_files
+        bash_command: |
+          mkdir -p $data_dir/us-counties
+          curl -o $data_dir/us-counties/us-counties-2020.csv -L $us_counties_2020
+          curl -o $data_dir/us-counties/us-counties-2021.csv -L $us_counties_2021
+          curl -o $data_dir/us-counties/us-counties-2022.csv -L $us_counties_2022
+          mkdir -p $data_dir/us-states
+          curl -o $data_dir/us-states/us-states.csv -L $us_states
+          mkdir -p $data_dir/excess-deaths
+          curl -o $data_dir/excess-deaths/excess-deaths.csv -L $excess_deaths
+          mkdir -p $data_dir/mask-use
+          curl -o $data_dir/mask-use/mask-use-by-county.csv -L $mask_use_by_county
+        env:
+          data_dir: /home/airflow/gcs/data/covid19-nyt
+          us_counties_2020: https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties-2020.csv
+          us_counties_2021: https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties-2021.csv
+          us_counties_2022: https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties-2022.csv
+          us_states: https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv
+          excess_deaths: https://raw.githubusercontent.com/nytimes/covid-19-data/master/excess-deaths/deaths.csv
+          mask_use_by_county: https://raw.githubusercontent.com/nytimes/covid-19-data/master/mask-use/mask-use-by-county.csv
+
+    - operator: GoogleCloudStorageToBigQueryOperator
+      description: Task to load the data from Airflow data folder (GCS Composer bucket) to BigQuery
+      args:
+        task_id: load_us_counties_to_bq
+        bucket: "{{ var.value.composer_bucket }}"
+        source_objects: ["data/covid19-nyt/us-counties/us-counties-*.csv"]
+        source_format: CSV
+        destination_project_dataset_table: covid19_nyt.us_counties
+        skip_leading_rows: 1
+        write_disposition: WRITE_TRUNCATE
+        schema_fields:
+          - name: date
```

Comment:
Is it possible to add descriptions to these column schemas?

Reply:
Done!